markwon/src/main/java/ru/noties/markwon/renderer/SpannableMarkdownVisitor.java

```diff
@@ -386,6 +386,22 @@ private boolean handleTableNodes(CustomNode node) {
         return handled;
     }
 
+    @NonNull
+    private CharSequence removeFromEnd(int start) {
+
+        // this method is not intended to be used by clients
+        // it's a workaround to support tables
+
+        final int end = builder.length();
+
+        // as we do not expose builder and do no apply spans to it, we are safe to NOT to convert to String
+        final Editable impl = new SpannableBuilder(builder, start, end);
```

Comment:
Why `Editable` here? Regular `CharSequence` should be just fine.

Reply:
`CharSequence` doesn't have `delete(`.

Reply:
Yeah, but we are deleting from `SpannableBuilder` here